packages/plugin-infra/src/types.ts

```diff
@@ -1,8 +1,9 @@
 import type React from 'react'
 import type { Option, Result } from 'ts-results'
-import type { TypedMessage, TypedMessageTuple, ScopedStorage } from '@masknet/shared-base'
+import type { TypedMessage, TypedMessageTuple, ScopedStorage, ProfileIdentifier } from '@masknet/shared-base'
 import type { Emitter } from '@servie/events'
 import type { Web3Plugin } from './web3-types'
+import type { AddressName } from '../../shared/node_modules/@masknet/web3-shared-evm'
```

Comment:
Wrong import path. I suggest you re-export `AddressName` from `@masknet/web3-shared-evm`